src/raven.js

```diff
@@ -1320,8 +1322,15 @@ Raven.prototype = {
         return httpData;
     },
 
+    _shouldBackoff: function() {
+        return this._backoffDuration && now() - this._backoffStart < this._backoffDuration;
+    },
 
     _send: function(data) {
+        if (this._shouldBackoff()) {
+            return;
```

Comment:
I'd suggest that we `console.log` or something that this is happening. That way we're not dropping things in the dark without any insight as to why if someone were to debug. Including ourselves.